sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -50,9 +60,11 @@ public ClientLogger(Class<?> clazz) {
      * Retrieves a logger for the passed class name using the {@link LoggerFactory}.
      *
      * @param className Class name creating the logger.
+     * @throws RuntimeException it is an error.
```

Comment:
Provide better descriptive text than this! What causes the error?

Reply:
It is not needed. Forgot to delete. 
Removed.